sycl/include/sycl/detail/cg.hpp

```diff
@@ -42,111 +42,16 @@ namespace detail {
 class event_impl;
 using EventImplPtr = std::shared_ptr<event_impl>;
```

Comment:
Suggest to also remove a note about handler from  `BKMs on avoiding changing ABI` section in sycl/doc/developer/ABIPolicyGuide.md